scripts/pack-distlib.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eux -o pipefail
+
+echo WORKING_DIRECTORY=$WORKING_DIRECTORY
+echo "Preparing nested lib/dist directories for publishing..."
+
+# Ensure the nested_lib_dist directory exists and navigate to it
+mkdir -p nested_lib_dist
+
+# Find all lib/dist directories, excluding those in node_modules
```

Comment:
Let's add more comments here and in the gitignore file about the logic that this is implementing.